### PR TITLE
Add routes to login or main app

### PR DESCRIPTION
**Pull request by cristianbgp**
Add routes to the login view or discussions view

### CODE REVIEW

File: src/Discussion.js [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
+import { useUser } from "./contexts/user";
+
+function Discussion({ getAllComments, id }) {
+  const { discussions, setDiscussions } = useDiscussion();
+  const { user } = useUser();
+  const discussion = discussions.find(discussion => discussion.id == id);
 
-function Discussion({ discussion, getAllComments }) {
   let comments = getAllComments(discussion.id);
+  console.log("GAA", id);
```
Comment (test0n3) [inline src/Discussion.js]: Use a more descriptive way to announce your results.

File: src/Discussion.js [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
 
   return (
     <>
       <p>{discussion.id}</p>
       <p>{discussion.date}</p>
-      <p>{discussion.author}</p>
+      <p>{discussion.author.username}</p>
```
Comment (test0n3) [inline src/Discussion.js]: Has the structure to the Discussions changed?

File: src/DiscussionForm.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  const { setDiscussions } = useDiscussion();
+  const { user } = useUser();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const title = event.target.elements["title-discussion"].value;
+    const body = event.target.elements["body-discussion"].value;
+    setDiscussions({
+      id: Date.now(),
+      date: Date.now(),
```
Comment (test0n3) [inline src/DiscussionForm.js]: I had the thought of saving the date in string form, but if we want to change the date display, having it in numeric form seems more reasonable.

File: src/Discussions.js [hunk 4]
```diff
@@ -1,18 +1,21 @@
 import React from "react";
```
Comment (test0n3) [inline src/Discussions.js]: We really need to:

- Separate the files in directories, or
- Use distinctive name for the files

File: src/DiscussionForm.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    });
+    handleDialogOff();
+  }
+
+  const styledButton = {
+    backgroundColor: "tomato",
+    border: "none",
+    cursor: "pointer",
+    borderRadius: "50%",
+    width: "calc(1.4em + 1vw)",
```
Comment (liannivin) [inline src/DiscussionForm.js]: Could you please tell me what are you trying do with `calc`?

Comment (liannivin) [inline src/DiscussionForm.js]: Could you please tell me what are you trying do with `calc`?

Comment (test0n3) [reply src/DiscussionForm.js]: Is a prefix to use math. In this case we are doing a subtraction between 1.4em and the available width of the viewport. to correctly set the width of the modal.

**Pull request merged**